src/main/java/com/ldtteam/structurize/client/gui/WindowTagTool.java

```diff
@@ -16,22 +17,33 @@
 import net.minecraft.core.BlockPos;
 import net.minecraft.world.level.Level;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
+
+import static com.ldtteam.structurize.api.constants.Constants.GROUNDLEVEL_TAG;
 
 public class WindowTagTool extends AbstractWindowSkeleton
 {
+    /**
+     * List of tag options. Mods can just insert on this in mod constructor.
+     */
+    public static List<String> TAG_OPTIONS = new ArrayList<>();
```

Comment:
If the intention is that other mods should be writing into this list, maybe for safety make it a concurrent list?

Reply:
(Also probably want a method to add into it, to prevent mods from also clearing the list)

Reply:
Fair enough